### ISSUE TITLE
Is there any plan for building arm64 docker image

### ISSUE THREAD

**Issue by yuzp1996** [travelaudience/nexus-proxy#25]

**Comment by jeff-knurek** [comment]
no, there's no plan (same as https://github.com/travelaudience/docker-nexus/issues/37 )

We're using quay.io to build the image based on tag pushes to github. So the most direct way to accomplish this would be to add an addition Dockerfile to the repo, however that would then require maintaining two nearly identical Dockerfiles, not an approach we'd want to take. If you want to take ownership of building/testing a solution to create a second image, and submit the PR, we can adjust the build process in quay.io to make it happen.
